singer-connectors/tap-mongodb/requirements.txt

```diff
@@ -1 +1,2 @@
-pipelinewise-tap-mongodb==1.2.0
+# We need unreleased support for "srv" property in order to sync with Atlas
```

Comment:
Will release this new feature

Reply:
@elephantum 1.3.0 is out:
https://pypi.org/project/pipelinewise-tap-mongodb/1.3.0/

Reply:
Please try it out.